CHANGELOG.md

```diff
@@ -32,6 +33,52 @@
 - [0.10.0](#0100---20170307)
 - [0.9.9 and prior](#099---20170202)
 
+
+## [1.4.1]
+
+> Released 2019/12/03
+
+This is a patch release in the 1.4 series, and as such, strictly contains
+bugfixes. There are no new features nor breaking changes.
+
+### Fixes
+
+##### Core
+
+  - Fixed a memory leak in the balancer
+    [#5229](https://github.com/Kong/kong/pull/5229) --
+    Thanks [zeeshen](https://github.com/zeeshen) for the patch!
+  - Removed arbitrary limit on worker connections.
+    [#5148](https://github.com/Kong/kong/pull/5148)
+  - Fixed `preserve_host` behavior for gRPC routes
+    [#5225](https://github.com/Kong/kong/pull/5225)
+  - Fix migrations for ttl for OAuth2 tokens
+    [#5253](https://github.com/Kong/kong/pull/5253)
+  - Improve handling of errors when creating balancers
+    [#5284](https://github.com/Kong/kong/pull/5284)
+
+##### CLI
+
+  - Fixed an issue with `kong config db_export` when reading
+    entities that are ttl-enabled and whose ttl value is `null`.
+    [#5185](https://github.com/Kong/kong/pull/5185)
+
+##### Admin API
+
+  - Various fixes for Admin API behavior
+    [#5174](https://github.com/Kong/kong/pull/5174),
+    [#5178](https://github.com/Kong/kong/pull/5178),
+    [#5191](https://github.com/Kong/kong/pull/5191),
+    [#5186](https://github.com/Kong/kong/pull/5186),
```

Comment:
`,` extra comma at the end.
```suggestion
    [#5186](https://github.com/Kong/kong/pull/5186),´
```

Reply:
@bungle Thanks! Do you have any suggestion on how to better describe those various fixes? Some of them catch different edge-cases, so it was hard to summarize without getting too technical on the details. Or do you think this general description plus the PR links is fine?